Lesson4/Controllers/User profile/EditProfileViewController.swift

```diff
@@ -0,0 +1,22 @@
+import UIKit
+
+class EditProfileViewController: UIViewController {
+    
+    // MARK: - IBOutlets
+    @IBOutlet weak var descriptionLabel: UILabel!
+    @IBOutlet weak var statusTextView: UITextView!
+    @IBOutlet weak var saveButton: UIButton!
+    
+    // MARK: - properties
+    var statusEditDelegate: StatusEditDelegate?
```

Comment:
```suggestion
    weak var statusEditDelegate: StatusEditDelegate?
```
цикл сильных ссылок если это делегат